calendar/calendar-tool/tool/src/java/org/sakaiproject/calendar/tool/CalendarAction.java

```diff
@@ -3279,10 +3285,27 @@ protected void buildMonthContext(VelocityPortlet portlet,
 		
 		context.put("selectedView", rb.getString("java.bymonth"));
 		
+		context.put("isDefaultView", isDefaultView(state, ToolManager.getCurrentPlacement()));
+		
 		context.put("dayOfWeekNames", calObj.getCalendarDaysOfWeekNames(false));
 		
+		context.put("monthInt", Integer.valueOf(calObj.getMonthInteger()));
 	} // buildMonthContext
 	
+	protected boolean isDefaultView(CalendarActionState state, Placement currentPlacement)
+	{
+		String currentView = state.getState();
+		boolean isDefaultView = false;
+		String defaultView = currentPlacement.getPlacementConfig().getProperty( PORTLET_CONFIG_DEFAULT_VIEW);
+		
+		if (StringUtils.compare(defaultView, currentView) == 0)
+		{
+			isDefaultView = true;
+		}
+		
+		return isDefaultView;
```

Comment:
You can just return StringUtils.equals(defaultView, currentView) here

Reply:
Thanks for the suggestion, @plukasew!